src/ILCompiler.Compiler/src/Compiler/DependencyAnalysis/INodeWithDebugInfo.cs

```diff
@@ -19,12 +23,36 @@ public DebugLocInfo(int nativeOffset, string fileName, int lineNumber, int colNu
             ColNumber = colNumber;
         }
     }
-
+    
     public interface INodeWithDebugInfo
     {
         DebugLocInfo[] DebugLocInfos
         {
             get;
         }
+
+        DebugVarInfo[] DebugVarInfos
+        {
+            get;
+        }
+    }
+
+     
+    public struct DebugVarInfo
+    {
+        public uint varNumber;
```

Comment:
Can you capitalize the names and spell them out fully?
Ideally also make them `readonly`.